src/build-scripts/ci-startup.bash

```diff
@@ -73,4 +73,4 @@ elif [[ "${RUNNER_OS}" == "macOS" ]] ; then
 fi
 
 # Save the env for use by other stages
-src/build-scripts/save-env.bash
+# src/build-scripts/save-env.bash
```

Comment:
I think this line is going to turn out to be important.

Reply:
Fixed